sql/2021/structured-data/error_logs.sql

```diff
@@ -0,0 +1,13 @@
+#standardSQL
+  # A list of the error logs created in the run
```

Comment:
@jono-alderson is this only useful for metadata about the success of the custom metric itself, or is this intended to be a reader-facing metric? If the former we can omit it from the list of queries.

If there are any opportunities to fix bugs reported in the logs, please file an issue to track it or PR to fix it.